src/containers/guest-home-page/signup-form/SignupForm.js

```diff
@@ -12,6 +14,20 @@ const SignupForm = ({ handleSubmit, handleChange, handleBlur, data, errors }) =>
   const { inputVisibility: confirmPasswordVisibility, showInputText: showConfirmPassword } = useInputVisibility(
     errors.confirmPassword
   )
+  const { loading } = useSelector((state) => state.appMain)
```

Comment:
Interesting, why submit button should know about "appMain" loading state?

Reply:
currently, we are using redux for logging in and signing up and when we press the login or signup button it sets loading: true in appMain state